program-analysis/echidna/basic/working-with-eth.md

```diff
@@ -0,0 +1,119 @@
+# Using ether during a fuzzing campaign
+
+**Table of contents:**
+
+- [Using ether during a fuzzing campaign](#using-ether-during-a-fuzzing-campaign)
+  - [Introduction](#introduction)
+  - [Controlling the amount of ether in payable functions](#controlling-the-amount-of-ether-in-payable-functions)
+  - [Controlling the amount of ether in contracts](#controlling-the-amount-of-ether-in-contracts)
+  - [Summary: Working with ether](#summary-working-with-ether)
+
+## Introduction
+
+We will see how to use ETH during the fuzzing fuzzed. The following smart contract will be used as example:
```

Comment:
- we should be consistent in our naming of ETH/eth/ether through this section. Maybe let's use ether everywhere?
- "the fuzzing fuzzed" is probably not what you meant

Reply:
Fixed